examples/Scenario4Bundle.json

```diff
@@ -0,0 +1,76 @@
+{
+  "resourceType": "Bundle",
+  "type": "collection",
+  "entry": [
+    {
+      "fullUrl": "resource:0",
+      "resource": {
+        "resourceType": "Patient",
+        "name": [
+          {
+            "family": "Youngster",
+            "given": [
+              "Janet",
+              "B."
+            ]
+          }
+        ],
+        "birthDate": "2016-03-20"
+      }
+    },
+    {
+      "fullUrl": "resource:1",
+      "resource": {
+        "resourceType": "Immunization",
+        "status": "completed",
+        "vaccineCode": {
+          "coding": [
+            {
+              "system": "http://hl7.org/fhir/sid/cvx",
+              "code": "94"
+            }
+          ]
+        },
+        "patient": {
+          "reference": "resource:0"
+        },
+        "occurrenceDateTime": "2017-05-10",
+        "performer": [
```

Comment:
It looks like the "performer" and "lotNumber" elements didn't make it into the JWS payload or QR code

Reply:
Thanks for catching this -- may be an issue with the underlying configuration in the Ruby library, or the way I'm invoking it here. I can investigate but probably good if the mitre team can address https://github.com/dvci/health_cards/issues/84 first (there may be an issue with how "allowed properties" are inherited by HealthCare subclasses, or with how I have supplied these).

Reply:
Good catch @jdkizer9. I think this has to be fixed in the Ruby library itself, but we can put in a temporary fix in the examples script.